modules/cmdb/dao/issue_panel.go

```diff
@@ -65,17 +65,12 @@ func (client *DBClient) GetPanelByIssueID(issueID int64) (*IssuePanel, error) {
 }
 
 // 通过PanelID获取看板下的全部issue
-func (client *DBClient) GetPanelIssuesByPanel(panelID int64, pageNo, pageSize uint64) ([]IssuePanel, uint64, error) {
+func (client *DBClient) GetPanelIssuesByPanel(panelID int64) ([]IssuePanel, uint64, error) {
```

Comment:
should not remove pagination params, it will make crash in extreme case. I suggest do scroll at frontend.

Reply:
It should get all issues by panel as a full query map before issue page query.